parser/prism/Translator.cc

```diff
@@ -454,4 +509,44 @@ std::unique_ptr<parser::Node> Translator::translate(const Node &node) {
     return translate(node.get_raw_node_pointer());
 }
 
+std::unique_ptr<parser::Node> Translator::translateHash(pm_node_t *node, pm_node_list_t elements,
+                                                        bool isUsedForKeywordArguments) {
+    pm_location_t *loc = &node->location;
+
+    auto prismKVPairs = absl::MakeSpan(elements.nodes, elements.size);
```

Comment:
Not 100% sure this works with `{ **foo }`, maybe a comment since it's not all key-value pairs?

Reply:
Ah yeah, that's a `PM_ASSOC_SPLAT_NODE`... it's currently not implemented, but this should be handled okay as-is (since it calls into the `translate` method to decide how to handle it).

The variable name would be wrong in this case. I'll fix that.